### ISSUE TITLE
mss510x switch will not be updated in HA any longer when triggered through a other application

### ISSUE THREAD

**Issue by AlexanderTurnowsky** [krahabb/meross_lan#457]
## Version of the custom_component
5.2.1

## Configuration

```yaml

Dieser Fehler wurde von einer benutzerdefinierten Integration verursacht

Logger: custom_components.meross_lan.mss510x_###############################1
Quelle: custom_components/meross_lan/helpers/__init__.py:289
Integration: Meross LAN (Dokumentation, Probleme)
Erstmals aufgetreten: 06:40:53 (8 Vorkommnisse)
Zuletzt protokolliert: 09:39:45

AttributeError('list' object has no attribute 'get') in NamespaceHandler(Appliance.Control.ToggleX)._handle_dict: payload={'togglex': [{'channel': 0, 'onoff': 1, 'lmTime': 1718091516}]}
AttributeError('list' object has no attribute 'get') in NamespaceHandler(Appliance.Control.ToggleX)._handle_dict: payload={'togglex': [{'channel': 0, 'onoff': 0, 'lmTime': 1718091526}]}
AttributeError('list' object has no attribute 'get') in NamespaceHandler(Appliance.Control.ToggleX)._handle_dict: payload={'togglex': [{'channel': 0, 'onoff': 0, 'lmTime': 1718091581}]}
AttributeError('list' object has no attribute 'get') in NamespaceHandler(Appliance.Control.ToggleX)._handle_dict: payload={'togglex': [{'onoff': 1, 'lmTime': 1718091582, 'channel': 0}]}
AttributeError('list' object has no attribute 'get') in NamespaceHandler(Appliance.Control.ToggleX)._handle_dict: payload={'togglex': [{'channel': 0, 'onoff': 0, 'lmTime': 1718091584}]}

```

```

## Describe the bug
When i'm trigger the switch trough the Meross app or Alexa in HA the switch will not be updated. 
Doesn't matter if on or off is triggered in both direction it will not be updated.


## Debug log

<!-- To enable debug logs check this https://www.home-assistant.io/components/logger/ -->

```text

Add your logs here.

```

**Comment by krahabb** [comment]
Yep..this is originated from a bug which also affects #439. There's already a fix in the `dev` branch which I guess I should release asap..

Stay in touch!

**Comment by AlexanderTurnowsky** [comment]
Great this is fixed in Moonlight 2.2, all is working again 👍
